packages/frontity-org-theme/src/components/styles/global-styles.tsx

```diff
@@ -114,6 +114,16 @@ const elementBase = (colors: FrontityOrg["state"]["theme"]["colors"]) => css`
   }
   ul {
     list-style-type: "→ ";
+
+    & > li {
+      margin: 1.1rem 0;
```

Comment:
This margin doesn't match the design. According to it, the margin would be 16px, which equals `margin: 1.125rem 0`.